getcss.js

```diff
@@ -6,113 +6,18 @@ var _ = require('lodash')
 var filesize = require('filesize')
 var cssstats = require('cssstats')
 var getcss = require('get-css')
-
+var urls = require('./urls.json')
 
 var options = {
   timeout: 5000
-};
-
-var github = getcss('http://github.com', options)
-  .then(function(response) {
-    fs.writeFileSync('./data/github.css', response.css)
-  })
-  .catch(function(error) {
-     console.log("Can't get css because" + error)
-  });
-
-var pinterest = getcss('https://pinterest.com', options)
-  .then(function(response) {
-    fs.writeFileSync('./data/pinterest.css', response.css)
-  })
-  .catch(function(error) {
-     console.log("Can't get css because" + error)
-  });
-
-var pinterest = getcss('http://mapbox.com', options)
-  .then(function(response) {
-    fs.writeFileSync('./data/mapbox.css', response.css)
-  })
-  .catch(function(error) {
-     console.log("Can't get css because" + error)
-  });
+}
 
-var bluebottlecoffee = getcss('http://bluebottlecoffee.com', options)
-  .then(function(response) {
-    fs.writeFileSync('./data/bluebottlecoffee.css', response.css)
+urls.forEach(function (url) {
+  getcss(url, options).then(function (response) {
+    var domain = url.replace(/.*?:\/\//g, '')
```

Comment:
Strips the protocol. I'll create a new PR later to use an npm module rather than this regex magic.